submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,272 @@
+ caption='Processing rocket fuel into nuclear fuel'
+
+%}
+
+We see that nuclear fuel offers an impressive energy output of 1.21GJ, which is [a reference](https://wiki.factorio.com/Nuclear_fuel), but also around 10 times as much energy as the solid fuel first used to make the rocket fuel. This means that if you were to switch from burning coal directly to making it into nuclear fuel to burn instead, your coal demand would drop to a tiny fraction, to as little as 1/20 as before. 
+
+## Making the most of your uranium
+
+Despite its massive energy gains and the resulting coal and oil savings, nuclear fuel might not be perfect. The uranium-235 that it uses could instead be used to make uranium fuel cells, which provide a different energy situation:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing uranium-235 into uranium fuel cells'
+ caption='Processing uranium-235 into uranium fuel cells'
+
+%}
+
+We see that while uranium fuel cells can only be used inside nuclear reactors, they produce energy on the order of GJ instead of MJ, and the energy yield is not simply boosted but multiplied via the reactor neighbor bonus. Hence, up to hundreds of times more energy is gained from the same piece of uranium-235 if used in a nuclear reactor instead of nuclear fuel. This would suggest that if one has a uranium-235 supply already, they should just switch to nuclear power and electric machines instead of burner devices. However, it is also worth noting that nuclear fuel is unlocked alongside Kovarex Enrichment, which allows for easy production of uranium-235. This means that using uranium efficiently stops being an issue around the time you unlock nuclear fuel.
+
+## So how useful is fuel processing in general?
+
+Our findings so far about energy gains only partially help us answer this question. I have identified a more comprehensive list of factors that you should consider when deciding whether to (further) process your burner fuels:
+
+* **Gains in energy:** If you get more energy out of the processed fuel, then your demand for the unprocessed fuel decreases and so your resource patches run out more slowly. We determined the energy gains for different fuels earlier.
+- **Gains in vehicle acceleration bonus and top speed bonus:** Greater bonuses mean that your vehicles can go faster. More processed fuels offer greater bonuses.
+- **Gains in energy density (per stack and per unit):** More energy per stack means more vehicle range per slot of fuel, which makes a big difference for trains because of their limited fuel slots. Meanwhile, more energy per unit/stack can be interpreted as fewer units/stacks per energy, which means less logistics are required to move the same amount of energy. Needing fewer bots, belts, and/or trains to move fuels simplifies things and it also makes the logistic system itself cost less energy. More processed fuels have greater energy density in both regards, hence having these benefits.
```

Comment:
> which makes a big difference for trains because of their limited fuel slots

Most bases automatically insert fuel at the unloading station though, so I don't think it makes a difference for trains. It matters a little bit more for cars and tanks maybe, but they don't use that much fuel, and I personally manage to remember that they have to be refueled when on a journey by bringing some fuel in my/their inventory.

Reply:
I did originally mean locomotive slots. Good point that they can travel massive distances even with coal as long as there is a reasonable setup for refueling. I guess this energy density advantage does not matter so much. Ditto about cars and tanks.

The point about wagons I made later in the paragraph but I can explain it better.